tests/gen_client_server_certs_key_files.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+# #############################################################################
+# Simple shell script to generate PEM-certificates and private-key files
+# using the 'openssl' utility. These files are needed by the 
+# test_client_server_mtls.py test, which simulates a client-server SSL
+# channel with mutual TLS authentication.
+# #############################################################################
+
+set -Eeuo pipefail
+
+# Setup script globals, to establish curdir and root path to Certifier code base
+Me=$(basename "$0")
+
+# Establish directory locations
+pushd "$(dirname "$0")" > /dev/null 2>&1
+
+# shellcheck disable=SC2046
+CERT_ROOT="$(dirname $(pwd))"
+
+popd > /dev/null 2>&1
+
+TEST_DATA=${CERT_ROOT}/tests/pytests/data
+
+pushd "${TEST_DATA}" > /dev/null 2>&1
+
+# ----------------------------------------------------------------------------
+# NOTE: The -subj "/C=US/ST=CA/CN=test" clause ends up creating a certificate
+#       with only few fields (countryName, stateOrProvinceName, commonName) populated.
+#
+# This is why in the pytest we have the following to denote the 'CN=test'
+# fragment specifying the commonName: COMMON_NAME_FIELD = 2
+
+echo "${Me}: Generating self-signed server public certificate and private-key ..."
+openssl req -batch -new -x509 -days 365 -noenc \
+            -out server.public-cert.pem \
+            -keyout server-private.key \
+            --addext 'subjectAltName=IP:127.0.0.1' \
+            -subj "/C=US/ST=CA/CN=test"
+
+echo "${Me}: Generating self-signed client public certificate and private-key ..."
+openssl req -batch -new -x509 -days 365 -noenc \
+            -out client.public-cert.pem \
+            -keyout client-private.key \
+            --addext 'subjectAltName=IP:127.0.0.1' \
+            -subj "/C=US/ST=CA/CN=test"
+
+# ----------------------------------------------------------------------------
+# Steps to generate root certificate and then generate server / client certs
```

Comment:
HI, @rgerganov -- Here onwards, stuff is new in this bash script.

I've tried to introduce generating root cert (policy cert) and have built test cases to use them. 
(The test case still does not work; see my comments later on.)